src/pages/HomePage/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { MdSearch } from 'react-icons/md';
+
+import {
+	Container,
+	SearchBar,
+	Empty,
+	Illustration,
+	Title,
+	Subtitle,
+	Results,
+} from './styles';
+import Logo from '../../components/Logo';
+import MovieCard from './MovieCard';
+
+import { searchMovies } from '../../services/api';
+
+export default function HomePage() {
+	const [results, setResults] = useState([]);
+	const [value, setValue] = useState('');
+
+	useEffect(() => {
+		if (value === '') {
+			setResults([]);
+			return;
+		}
+
+		searchMovies(value).then(({ data }) =>
```

Comment:
Make sure to only request to the server when the user is ready. I mean, it's requesting on every `keydown` event, so there is a lot of unnecessary requests here. For instance, I've searched for `007` and the network has 3 requests, but only the last one is helpful. Did you get my point?

<img width="248" alt="Screenshot 2020-04-08 at 11 09 01" src="https://user-images.githubusercontent.com/4838076/78772250-63cbc300-7989-11ea-9c75-a05cbed3fc95.png">

To solve it, there are some common approaches to do it, like [debounce and throttle](http://loopinfinito.com.br/2013/09/24/throttle-e-debounce-patterns-em-javascript/). Try to think which one fits better for this problem.